Determine why requests to core.trac.wordpress.org are hitting a rate limit

The core.trac.wordpress.org job is hitting a rate limit and causing the workflow to fail: https://github.com/johnbillion/rave-wordpress/actions/workflows/rave.yml

```
Run wget -O package.zip "https://core.trac.wordpress.org/browser/tags/6.7?format=zip"
--2024-11-17 22:03:28--  https://core.trac.wordpress.org/browser/tags/6.7?format=zip
Resolving core.trac.wordpress.org (core.trac.wordpress.org)... 198.143.164.246
Connecting to core.trac.wordpress.org (core.trac.wordpress.org)|198.143.164.246|:443... connected.
HTTP request sent, awaiting response... 429 Too Many Requests
2024-11-17 22:03:28 ERROR 429: Too Many Requests.

Error: Process completed with exit code 8.
```

--- a/.github/workflows/reproduce.yml
+++ b/.github/workflows/reproduce.yml
@@ -52,7 +52,7 @@ jobs:
       - name: Download zip from core.trac.wordpress.org
         if: inputs.source == 'core.trac.wordpress.org'
         run: |
-          wget -O package.zip "https://core.trac.wordpress.org/browser/tags/${{ steps.tag.outputs.short }}?format=zip"
+          wget -O package.zip "https://core.trac.wordpress.org/browser/tags/${{ steps.tag.outputs.short }}?format=zip" --user-agent="WordPress/${{ steps.tag.outputs.branch }}"
           unzip package.zip
           shopt -s dotglob
           mv ${{ steps.tag.outputs.short }}/* .

